pyartcd/tests/pipelines/test_prepare_release_konflux.py

```
@@ -932,6 +934,9 @@ async def test_run_exits_unstable_when_deferred_build_failures_exist(
         mock_verify_payload,
         mock_report_deferred_build_failures,
     ):
+        mock_registry_config.return_value.__enter__ = Mock(return_value='/tmp/fake-auth.json')
```

Ruff (0.15.10)

[error] Probable insecure usage of temporary file or directory: "/tmp/fake-auth.json"

(S108)